data/mods/PKs_Rebalance/pk_critter_insect.json

```diff
@@ -0,0 +1,410 @@
+            "cooldown" : 25,
+            "accuracy" : 10,
+            "damage_max_instance" : [
+                {"damage_type" : "stab", "amount" : 7, "armor_multiplier" : 0.3},
+                {"damage_type" : "bash", "amount" : 12, "armor_multiplier" : 0.3}
+            ]},
+        ["GRAB", 30], ["GRAB_DRAG", 15],
+        {
+            "type" : "leap",
+            "cooldown" : 15,
+            "max_range" : 2
+        }],
+    "description":"Somewhat smaller than other soldiers, this nimble ant jumps after prey and drags them back to the hive for processing.  Its mandibles are the size of its thorax, and secrete poison.",
+    "flags":["SEES", "SMELLS", "SWIMS", "CHITIN", "CLIMBS", "BADVENOM", "BASHES"],
+    "anger_triggers":["FRIEND_ATTACKED", "FRIEND_DIED", "HURT", "PLAYER_CLOSE", "PLAYER_WEAK"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_ant_soldier_pk",
+    "name": "jack ant",
+    "species":"INSECT",
+    "default_faction":"ant",
+    "symbol":"a",
+    "color":"magenta",
+    "size":"MEDIUM",
+    "material":"iflesh",
+    "diff":20,
+    "aggression":8,
+    "morale":80,
+    "speed":125,
+    "melee_skill":8,
+    "melee_dice":2,
+    "melee_dice_sides":7,
+    "melee_cut":8,
+    "dodge":3,
+    "armor_bash":18,
+    "armor_cut":10,
+    "armor_acid":3,
+    "armor_fire":2,
+    "vision_day":30,
+    "vision_night":5,
+    "hp":75,
+    "death_function":"NORMAL",
+    "special_attacks":[
+        {
+            "type" : "bite",
+            "cooldown" : 25,
+            "accuracy" : 10,
+            "damage_max_instance" : [
+                {"damage_type" : "stab", "amount" : 15, "armor_multiplier" : 0.2},
+                {"damage_type" : "bash", "amount" : 20, "armor_multiplier" : 0.2}
+            ]},
+        ["GRAB", 30], ["GRAB_DRAG", 15],
+        {
+            "type" : "leap",
+            "cooldown" : 15,
+            "max_range" : 3
+        }],
+    "description":"Somewhat smaller than other soldiers, this nimble ant jumps after prey and drags them back to the hive for processing.  Its mandibles are the size of its thorax, and secrete a slick-looking poison.",
+    "flags":["SEES", "SMELLS", "SWIMS", "CHITIN", "CLIMBS", "BADVENOM", "BASHES"],
+    "anger_triggers":["FRIEND_ATTACKED", "FRIEND_DIED", "HURT", "PLAYER_CLOSE", "PLAYER_WEAK"]
+    },{
+    "type" : "MONSTER",
+    "id" : "mon_ant_queen_firebug",
+    "name": "queen ant",
+    "species":"INSECT",
+    "default_faction":"ant",
+    "symbol":"a",
+    "color":"ltred",
+    "size":"LARGE",
+    "material":"iflesh",
+    "diff":18,
+    "aggression":0,
+    "morale":100,
+    "speed":40,
+    "melee_skill":4,
+    "melee_dice":3,
+    "melee_dice_sides":4,
+    "melee_cut":4,
+    "dodge":1,
+    "armor_bash":16,
+    "armor_cut":25,
+    "armor_acid":5,
+    "vision_day":5,
+    "vision_night":5,
+    "hp":140,
+    "death_function":"NORMAL",
+    "special_attacks":[["ANTQUEEN", 3], ["FLAMETHROWER", 10], ["SHRIEK_ALERT", 15]],

```

Comment:
Ant queens, as insects, are afraid of own fire. I don't know if it's possible to redefine species, but if possible, it could help.